build.sbt

```diff
@@ -659,9 +659,9 @@ lazy val graph = (project in file("lib/scala/graph/"))
   .configs(Test)
   .settings(
     version := "0.1",
-    resolvers ++= Seq(
-      Resolver.sonatypeRepo("releases"),
-      Resolver.sonatypeRepo("snapshots")
+    resolvers ++= (
+      Resolver.sonatypeOssRepos("releases") ++
```

Comment:
Does this introduce any additional repos? What are they for?

Reply:
API was deprecated and replaced with this one